main.go

```diff
@@ -63,6 +63,11 @@ func init() {
 
 	// Set global trace flag.
 	globalTrace = os.Getenv("MINIO_TRACE") == "1"
+
+	switch os.Getenv("MINIO_DEBUG") {
```

Comment:
In future MINIO_DEBUG can have multiple debugging options, `lock`, `health` and so on... so placed it in switch case statment.